### DISCUSSION TITLE
Login

### DISCUSSION THREAD

**Discussion by finnie2006** [prometheus/prometheus#9435] [Q&A] [answered]
What?
A login system
Why? 
if u run promeheus for instance on an vps in an docker container u cant protect it
i dont understand why this feature isnt present

**Comment by roidelapluie** [comment, marked as answer]
You can setup users and TLS to secure your Prometheus instance:

https://prometheus.io/docs/prometheus/latest/configuration/https/

See also https://inuits.eu/blog/prometheus-server-tls/

**Reply by finnie2006** [reply]
is this also possible when running in docker?

**Reply by roidelapluie** [reply]
Yes, you just need to provide the extra configuration file in your volume and the proper flag for prometheus to use it.
